contrib/init/systemd/balena-engine.service

```diff
@@ -10,25 +10,37 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/balena-engine-daemon -H fd://
+ExecStart=/usr/local/bin/balena-engine-daemon -H unix:///var/run/balena-engine.sock
```

Comment:
This would obsolete the `balena-engine.sock` unit.

I tested this for a bit and it seems we run into https://github.com/moby/moby/issues/30348 with this solution

Reply:
@ofirule if you can revert the change to the socket location I can merge this if you are still interested...